### DISCUSSION TITLE
Add In Column For Is Logged In For Users

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#15906] [Ideas]
I know there's an Is Active and Last Login columns and variables. But it would be great to see and and add a user column for "Is Currently Logged In." The use case is that if one has to do maintenance, it would be great to know if users are currently logged in, so they can be notified.

**Comment by jeremystretch** [comment]
How would you determine whether a user is currently logged in? The only information NetBox has is when the user last made an authenticated request (the "last active" time).

**Reply by xkilian** [reply]
That is good enough. At least, we can ping those people directly to make sure they are finished their tasks if they are admins. Otherwise the maintenance banner should change be ok.

**Reply by mr1716** [reply]
Maybe that's the value that gets added to the users table. Just thinking that the last time the user made an authenticated request would be enough.

**Comment by mr1716** [comment]
> That is good enough. At least, we can ping those people directly to make sure they are finished their tasks if they are admins. Otherwise the maintenance banner should change be ok.


That works great when there are a few people using the tool in the same or non-overlapping timezones. However, for larger teams + instances and/or in instances where people work overlapping timezones (such as EST and Hawaii or US and APAC), it may make sense to have that 1 column added in. It has the potential to reduce workload and save time of having to ping the people.
